user_defined.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+# This script allows your recordings to be transcoded again when ABR is enabled for the bitrate you specify.
```

Comment:
Please give enough information to let the user know why he needs to use this script or under which circumstances he needs to use this script

Reply:
I'm also copying the comments in the conversation here
```
Could you add issue number(if exists) and make the name of file more desriptive about its function?
Why have we developed this script?
user_defines_script is so generic and I don't understand what it does from its name.
```